charts/pomerium/templates/secret.yaml

```diff
@@ -53,10 +53,15 @@ stringData:
 {{- end -}}
 
 {{- end -}}
+{{- if .Values.config.insecure }}
```

Comment:
Can we move this into a named template in `_helpers.tpl` and reference that with `include`?  I don't want to be setting inline in a template.

It should also be an `internalScheme` or something.  The scheme for things like the authenticate service and an external forward auth endpoint should be independent.  Also see other comment on this.